docs/changes.rst

```diff
@@ -31,6 +31,20 @@ Released: not yet
   surrounded by verbose test (if p.parse.verbose:). See pywbemcli issue
   #395,
 
+* Fixes bugs in SubscriptionManager where 1. host_name  of
+  CIM_ListenerDestination and CIM_Filter objects inconsistently used
```

Comment:
It is not CIM_Filter but CIM_IndicationFilter

Reply:
DONE

Reply:
I don't see this as done.

Reply:
DONE again